### ISSUE TITLE
Fan Speed Control

### ISSUE THREAD

**Issue by iTheGentle** [ThomasVon2021/pikvm-CM4-Board#18]
Hello,
First of all i want to Thank you for the best kvm solution ever.
but i faced a problem with fan sound it's annoying, will modifying config file like this will help to switch fan on while the temperature is high only, and off it while the temperature is normal?
here what i found when i googled about this issue:

```
$ sudo vi /boot/config.txt
# PoE Hat Fan Speed Control
dtoverlay=rpi-poe
dtparam=poe_fan_temp0=57000
dtparam=poe_fan_temp1=60000
dtparam=poe_fan_temp2=63000
dtparam=poe_fan_temp3=66000
```
thank you in advance.

**Comment by hzzfly** [comment]
The fan does not support speed control now. You can choose passive cooling method with heat dissipation stickers.

**Comment by PhilipTS** [comment]
@ThomasVon2021 maybe in a next HW revision you could implement the fan control (PWM) like it is in the HATv3?

As the fan was to loud I cut a piece of aluminium and put it between the case and the CPU with some thin thermal pads.
(2cm x 4cm x 2.5mm)
It keeps the CPU temps low without the fan.

**Comment by ThomasVon2021** [comment]
Yes, we have alreadly test fan control ok in next version. We will keep this issue until we publish new HW.

**Comment by iTheGentle** [comment]
Thats good, would you please give us (owners of old version) a "how to" to upgrade our hw with fan control, tbh we will never buy a new hw only for this feature.

**Comment by m50S79sM6SRNp8Jn** [comment]
[BliKVM discord @https://discord.gg/dwx7vKYt](https://discord.gg/dwx7vKYt)
